templates/amt/hovedvedtak.hbs

```diff
@@ -41,10 +41,10 @@
             {{/if}}
         </section>
 
-        {{#if deltaker.bakgrunnsinformasjon}}
+        {{#if this.deltaker.bakgrunnsinformasjon}}
```

Comment:
Hva er det man løser med `this` her, hvis det ikke trengs på de andre `deltaker.*`?

Reply:
Ja, det vises i gosys, så jeg tenkte å teste om dette funka i mangel på andre ideer 🤔